frontend/src/ModelOverview.tsx

```diff
@@ -69,21 +71,53 @@ function ModelOverview({ version }: ModelOverviewProps) {
             </Box>
           </Box>
           <Box sx={{ p: 2 }}>
-            <Typography variant='h6'>{schema?.schema.properties.contacts.properties.uploader.title}</Typography>
+            <Stack direction='row'>
```

Comment:
The other Stacks have a spacing prop, does this one not need it?

Reply:
I mistakenly put the closing typography tag on line 72, a bit too far down. So when it came to spacing, it was spacing everything inside.